lib/rack/tracker/facebook/facebook.rb

```diff
@@ -12,7 +12,7 @@ def render
   end
 
   def self.track(name, *event)
-    { name.to_s => [event.last.merge(class_name: 'Event')] }
+    { name.to_s => [event.last.deep_stringify_keys.merge('class_name' => 'Event')] }
```

Comment:
instead of repeating this in every handler it should be done where the event is written in the env